src/view/components/switch/CustomSwitch.module.scss

```diff
@@ -1,37 +1,60 @@
 .switch {
-    background-color: #f5fcff;
     height: 1.85rem;
-    width: 15rem;
+    width: 170px;
     position: relative;
     border-radius: 0.285rem;
-    text-align: center;
     font-size: 0.8rem;
+    display: flex;
+    align-items: center;
+    margin-inline-end: -20px;
+
+    &.checked {
+        margin-inline-start: -20px;
+        margin-inline-end: 0;
+
+        .tag {
+            background-color: #4e88c7;
+            color: white;
+            left: 150px;
+        }
+
+        .label {
+            background-color: #74adf0;
+            color: white;
+        }
+    }
+
```

Comment:
Could we make the size of the switch responsive? It looks out of place on a mobile phone.

Reply:
Looks better this way?
![image](https://github.com/talyaron/delib-5-p/assets/25402124/9afc4060-8827-4969-9126-e627749e4b5d)

Reply:
I didn't make it responsive to the level of changing size when window size changes, but I fixed my negative margins and now it's aligned better

Reply:
I do think we should make it smaller if the screen size becomes very small, but I will let @talyaron make that call :)